javascript-forms/form-values.js

```diff
@@ -0,0 +1,13 @@
+var $contact = document.getElementById('contact-form');
+var elements = {};
```

Comment:
I would move this inside the function. It is better to create this object as needed.